developer-guide.md

```diff
@@ -243,42 +245,10 @@ token. If available, the choices are:
     typically less than 5%.
 
-<a name="valid-until"></a>
-### 21. What does the _Valid Until_ date mean for my tokens?  
-
-- Tokens are guaranteed to be accepted by Chrome up to the _Valid Until_ date. This date is calculated
-    based on your feedback, initially set to 6 weeks after the token is created.
-- Feedback is required every 6 weeks to extend the _Valid Until_ date in order to keep
-    participating in the trial. 
-- Before the _Valid Until_ date arrives, we'll send an email to invite you to provide feedback
-    and continue participating in the trial.
-- Tokens without any feedback in the past 6 weeks will automatically be disabled by a remote
-    update process after a grace period.
-- We will send a last warning email before disabling the token. This will be your last chance
-    to provide feedback and keep the token active. After that your token will no longer be accepted by
-    Chrome, and will not enable the experimental feature on any page.
-- You can still provide feedback after the token is disabled. However, the token will not be
-    re-enabled immediately, even though the _Valid Until_ date is updated. The process to disable
-    and re-enable tokens runs periodically, and it takes some time for the remote updates to
-    reach the majority of Chrome clients.
-- We issue tokens this way for a number of reasons, most importantly:
-  - To prevent experimental features from becoming "burned in" to the web platform. With
-      shorter-lived tokens, we can ensure that no site can use a feature for more than a month or
-      two, without checking in with us.
-  - To provide an opportunity to collect feedback on features from web developers. Feedback is
-      required to extend the lifespan of the token.
-  - To ensure active trial participants can benefit from the trial without the overhead of
-      repeatedly deploying new tokens.
-
-<a name="valid-until-feedback"></a>
-### 22. How do I keep my tokens active for the entire trial? 
-
-- On the registration page, you can provide feedback for the trial. Use the Feedback button to
-    complete a survey to provide feedback.
-- Providing feedback will extend the Valid Until date by 6 weeks. You will see the updated
-    _Valid Until_ date on the registration page after you submit the feedback. If you have
-    multiple origins registered for a trial, we'll only ask you for once per period, rather than
-    for every origin.
+<a name="expiry-date"></a>
+### 22. What does the expiry date mean for my tokens?  
+
+Tokens are guaranteed to be accepted by Chrome up to the expiry date. This date is typically identical with the end date of the origin trial. In some instances, the end date of the origin trial is extended from the trial's origin end date. If you would like to continue participating in the origin trial during this additional phase, you will need to generate a new token by selecting the "Renew" button on your registration's detail page.
```

Comment:
I know the existing question about _Valid Until_ uses the "guaranteed to be accepted" verbiage. However, that's not really true, given a trial/token can be disabled at any time. I suggested alternate verbiage, although most anything that avoids "guaranteed" would be fine.

Also, don't repeat the explanation about token expiry != trial end date. Instead, link to the existing explanation. I suggest adding an explicit anchor tag to question 11, and referring to that.


```suggestion
Tokens are considered valid by Chrome up to the expiry date. The expiry date is usually the same as the end date of the origin trial. See <link to question 11>.
```